var/spack/repos/builtin/packages/costa/package.py

```diff
@@ -13,32 +13,35 @@ class Costa(CMakePackage):
     Based on the paper: https://arxiv.org/abs/2106.06601
     """
 
-    maintainers("haampie", "kabicm", "RMeli")
+    maintainers("haampie", "kabicm", "RMeli", "mtaillefumier")
     homepage = "https://github.com/eth-cscs/COSTA"
+    url = "https://github.com/eth-cscs/COSTA/releases/download/v2.0/COSTA-v2.0.tar.gz"
```

Comment:
This was removed in #35137 in favour of `url_from_version` (see https://github.com/spack/spack/pull/35137#discussion_r1085678357).

Reply:
```suggestion
```